src/app/QuickStarts/quickstarts/automated-rules-quickstart.tsx

```diff
@@ -55,16 +55,13 @@ const AutomatedRulesQuickStart: QuickStart = {
   spec: {
     version: 2.3,
     displayName: 'Automated Rules',
```

Comment:
Lets update our quickstart titles, following this guideline:

https://docs.openshift.com/container-platform/4.12/web_console/creating-quick-start-tutorials.html#quick-start-content-guidelines-card-copy_creating-quick-start-tutorials

> Start with a verb and communicate the goal of the user.

Maybe for now, lets just go with `Get started with {some-feature}`.